.github/workflows/changelog.yml

```diff
@@ -1,40 +1,9 @@
 name: Check changelog file included
 
-on: workflow_call
+on: pull_request
```

Comment:
Thanks for opening the PR @jjmerchante 
Since we are only interested to check if there is a changelog entry in the code changes, I think we can add a rule to the workflow.

```
on:
  pull_request:
    paths:
      - 'release_tools/**'
```

What do you think?

Reply:
Thanks @vchrombie 

But in that case the action will only run when there is a new pull request and a file has been added to the release_tools.

If a changelog entry is not created in a pull request, it will not run and will not fail.

Reply:
Sorry I understood something different, I updated the code with your proposal.

Reply:
Thanks for adding it.

This rule will effectively avoid all the changes other than changes to the source code. Since we don't need a changelog for changes to tests, poetry changes, & documentation, we don't need the workflow to be triggered. Whenever a PR with a change in the `release_tools` folder is opened, the workflow should be triggered.